### PR TITLE
Vote24-1: Software Challenge (SPL Rules)

### PR DESCRIPTION
**Pull request by 16stelter**

### CODE REVIEW

File: Section_III/software-challenge.tex [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Participating teams have to create a poster (A3 or A2), and prepare a short 5 minutes oral presentation which additionally explains and shows the idea and results of this approach.
+If there is a monitor on site, it can be used for the presentation. 
+However, this depends on the location and the teams should be prepared to give their presentation only with their poster if necessary.
+
+\smallskip
+
+Teams are encouraged to publish the code for their approach to enable a fast progress in the league. 
+However, only the top 3 ranked teams are required to publish their code
+with instructions within one month after the RoboCup 2022.
```
Comment (MaikePaetzel) [inline Section_III/software-challenge.tex:21]: 2023?

File: Section_III/software-challenge.tex [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+\bigskip
+
+{\bfseries Scoring}
+\smallskip
+
+The winner will be decided by a vote among the team leaders using the Borda count mechanism\footnote{\url{http://en.wikipedia.org/wiki/Borda_count}}. 
+Each participating team will vote for their top 5 teams in order (excluding themselves).
```
Comment (MaikePaetzel) [inline Section_III/software-challenge.tex:28]: What if less than 5 teams participate?

File: Section_III/software-challenge.tex [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+Teams are encouraged to evaluate the presentations based on the following criteria:
+
+\begin{itemize}
+	\item Novelty
+	\item Benefit to the league
+	\item Documentation
+	\item Ease of transfer to other teams
+\end{itemize}
+
+At a time decided by the designated referee, within one hour of the last demonstration if not otherwise specified, the captain of each team will submit the team's rankings by filling out an online form.
```
Comment (MaikePaetzel) [inline Section_III/software-challenge.tex:38]: I would remove "online" here to allow us to use a printed form if we do not have a stable internet connection at the venue

**Comment by timonegk** [comment]
I do not like the name "Software Challenge" because it sounds like the TC is posing a challenge and all teams are requiring to solve it. But if I understand it correctly, the teams present recent research. I would suggest to rename it to something like "Software presentation" or "Research presentation".

**Comment by MaikePaetzel** [comment]
This rule change has been approved during the 2023 rule voting with for/against/abstain: 8/4/1

**Pull request merged**